### PR TITLE
Fixes to the documentation and added myself into the contributor section

### PR DESCRIPTION
**Pull request by HPOA909**
This seems appropriate when 4.0 was dpreciate 3-4 days ahead

### CODE REVIEW

**Comment by andrewdavidwong** [comment]
> This seems appropriate when 4.0 was dpreciate 3-4 days ahead

No, 4.0 has not reached EOL yet. Please do not submit PRs anticipating routine administrative/management actions. We will take care of it. If we somehow forget, than a PR would be appropriate.

File: _data/team.yml [hunk 1]
```diff
@@ -308,3 +308,10 @@
   role: "HCL page maintenance, HCL scripts"
   type: community
   fingerprint: 9AF3 388A 533E 73AD 1107  7E55 617D 7048 8F0E 7BC6
+
+- name: HPOA909 (Howard Chen)
+  email: hpoa.business@gmail.com
+  role: "Documentation modifier"
+  type: community
+  fingerprint: 
+  github: hpoa909
```
Comment (andrewdavidwong) [inline _data/team.yml:317]: This needs to be added so as to maintain the alphabetical order of the list.

File: _data/downloads.yml [hunk 2]
```diff
@@ -93,8 +93,8 @@ releases:
   Qubes release 4.0.4:
     testing: no
     latest: no
-    aging: yes
-    deprecated: no
+    aging: no
+    deprecated: yes
```
Comment (andrewdavidwong) [inline _data/downloads.yml:97]: Not true yet. See comment on this PR.

**Review by andrewdavidwong — CHANGES_REQUESTED** [review]

File: _data/downloads.yml (hunk 2 above)
Comment (HPOA909) [reply _data/downloads.yml:97]: I may wait til 4.0 depreciate

Comment (andrewdavidwong) [reply _data/downloads.yml:97]: > I may wait til 4.0 depreciate

Please do not submit PRs anticipating routine administrative/management actions. We will take care of it. If we somehow forget, than a PR would be appropriate.

**Comment by Minimalist73** [comment]
I don't get the point of this PR, @andrewdavidwong always do the edits in time so there is no need for this. Also, It seems weird to me that you add yourself to the contributor list just because you edited 2 lines on the whole website.

**Comment by HPOA909** [comment]
4.0 has blew from the EOL-land and merge of this may required.

**Comment by andrewdavidwong** [comment]
> 4.0 has blew from the EOL-land and merge of this may required.

It's already done.

**Pull request closed without merging**